openapi.yaml

```diff
@@ -333,6 +368,8 @@ components:
                             type: integer
                             description: Min time between consecutive messages. Milliseconds.
                             example: 3000
+                          html:
+                            type: boolean
```

Comment:
please add `description: If true, the messageBody value is treated as HTML content`